hw/ps4/liverpool/lvp_gfx.c

```diff
@@ -24,6 +24,7 @@
 #include "hw/ps4/liverpool/pm4.h"
 #include "hw/ps4/macros.h"
 #include "gca/gfx_7_2_d.h"
+#include "gca/gfx_7_2_sh_mask.h"
```

Comment:
is this used?

Reply:
It doesn't seem to be used, so it should be removed.